eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Utxo.hs

```diff
@@ -327,15 +321,26 @@ instance
 
   transitionRules = [utxoInductive]
 
-  renderAssertionViolation AssertionViolation {avSTS, avMsg, avCtx, avState} =
-    "AssertionViolation ("
-      <> avSTS
-      <> "): "
-      <> avMsg
-      <> "\n"
-      <> show avCtx
-      <> "\n"
-      <> show avState
+  renderAssertionViolation
```

Comment:
Wondering if these are part of the "API", so any chance someone could be parsing these failures? Should we mention them in the changelog?

Reply:
No renderAssertionViolation returns a string. It is not meant to return something parseable.